src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -54,7 +56,7 @@
 
     <Exec
       Condition="'$(OS)' != 'Windows_NT'"
-      Command="chmod a+x &quot;$(ToolRuntimePath)/corerun&quot;" />
+      Command="chmod a+x &quot;$(ToolRuntimePath)/CoreRun&quot;" />
 
```

Comment:
This should also be all lowercase. In fact we should probably just use the ToolHost property.